postgresql/postgresql_installer.sh

```diff
@@ -254,10 +254,10 @@ setup_standard_init()
         read yn
         if [ -z "`echo $yn | egrep -io '^n'`" ]; then
             CREATED_INIT="yes"
-            echo -n "log file: [/usr/local/copperegg/log/revealmetrics_postgresql.log] "
+            echo -n "log file: [/usr/local/copperegg/log/postgresql_metrics.log] "
```

Comment:
no changing of log filenames. lets use the same consistent naming-convention for memcached as well.